### DISCUSSION TITLE
Pixel art icons

### DISCUSSION THREAD

**Discussion by pukallus** [falling-fruit/falling-fruit-web#934] [Ideas]
I've always wanted to build this app.. turns out you guys were years ahead of me!

With DALL-E being able to generate pixel art, you could make the interface a bit more quirky and fun. When planning the app in my head I always thought pixel art would be a nice touch, a couple attached for reference. I have some python code which could help generate a pixel art image automatically to save manual intervention.

![bananas](https://github.com/falling-fruit/falling-fruit/assets/156563550/20b82096-d5f4-407b-82e6-2b6c3e296683)
![coconut](https://github.com/falling-fruit/falling-fruit/assets/156563550/c38a7480-bebe-4daa-88be-3eed38f760fd)
![guava](https://github.com/falling-fruit/falling-fruit/assets/156563550/77de244b-0e48-41c0-a750-7a64b8e98e90)
![mango](https://github.com/falling-fruit/falling-fruit/assets/156563550/80c506a7-d4d8-45e4-992f-d21ef327ff87)

**Comment by ezwelty** [comment]
@pukallus Neat! In principle, I love the idea. I agree that our interfaces could be more cheerful. Where do you envision the icons to be used? I don't think using these icons as pins on the map would work very well (?) because we have 3000 types of edible plants on the map – we've found text labels to be more informative. But I can see them as nice additions to for example the location window, the activity page, or the to-be-developed type page.

Would you be willing to share your Python code? Or try some of the Falling Fruit types (see the list [here](https://fallingfruit.org/data)), I guess as English common name and scientific name as input, and share the results? For example Blackthorn (_Prunus spinosa_) – a type of wild plum – to lavender (_Lavandula_), which is rather sought after for flower rather than fruit – to say the very specific McIntosh apple (_Malus pumila 'McIntosh'_).

**Comment by ezwelty** [comment]
A similar suggestion from Darin Daly (Ohio State University Commons Ecology 2025):

![design](https://github.com/user-attachments/assets/fd4594a3-b9af-4a51-8f03-a17ba133950a)

> Assigned icons to help users quickly understand some aspect of the plant; are its edibles easily recognizable one like an apple... is it large or small? I think it's okay to have a set of generic icons as its simplicity could get the job done. Very complex icons would be overwhelming.

Frankly I'm still not sold on the practicality of this idea of icons, because the text labels we already have are more informative. But I appreciate that this is because I know most of the plants. The suggestion of using iconography to highlight the main edible feature of a plant is interesting and could at least be used on future plant profiles to give a quick graphical overview of a plant's edible parts and/or uses.
